nmoscommon/auth/claims_options.py

```diff
@@ -1,90 +1,74 @@
 IS_XX_CLAIMS = {
-    "iat": {"essential": True},
-    "nbf": {"essential": False},
-    "exp": {"essential": True},
     "iss": {"essential": True},
     "sub": {"essential": True},
     "aud": {"essential": True},
-    "scope": {"essential": True},
-    "x-nmos-api": {"essential": True}
+    "exp": {"essential": True},
+    "iat": {"essential": False},
+    "nbf": {"essential": False},
+    "client_id": {"essential": True},
+    "scope": {"essential": False},
+    "x-nmos-api": {
+        "essential": True,
+        "values": ["registration", "query", "node", "connection"]
+    }
 }
 
 IS_04_REG_CLAIMS = {
```

Comment:
How do you envisage these being used? Is there no way to reduce the duplication a little, perhaps by copying the IS_XX_CLAIMS object and overriding part of it?